src/internal/modules/billing/controllers/bill-run.js

```diff
@@ -111,11 +116,20 @@ const getBillingBatchCancel = async (request, h) => billingBatchAction(request,
 
 const postBillingBatchCancel = async (request, h) => {
   const { batchId } = request.params;
+
+  try {
+    // Reset flag for rebilling on linked batches (before cancelling)
```

Comment:
This is not in the acceptance criteria for the ticket. The user might want to cancel the batch but keep the bill flagged for reissue.